tests/wallet/sync/test_wallet_sync.py

```diff
@@ -312,8 +312,9 @@ async def test_long_sync_wallet(self, two_wallet_nodes, default_1000_blocks, def
         for wallet_node, wallet_server in wallets:
             await time_out_assert(600, wallet_height_at_least, True, wallet_node, len(default_400_blocks) - 1)
 
+        reorg_blocks = bt.get_consecutive_blocks(num_blocks=400,block_list_input=default_400_blocks[:10])
```

Comment:
generating this many block is non-trival work. we have some test fixtures (that are pre-calculated) to form a reorg-chain that you might be able to use instead. It would speed up the test.

see `test_long_reorg_blocks`, which shared the first 320 blocks with `default_1500_blocks`

Reply:
cool thanks, i thought i would need to write one